core/src/main/java/de/danoeh/antennapod/core/feed/FeedItemFilter.java

```diff
@@ -20,6 +21,20 @@
     public final boolean showIsFavorite;
     public final boolean showNotFavorite;
 
+    static final String PLAYED = "played";
+    static final String UNPLAYED = "unplayed";
+    static final String NEW = "new";
+    static final String PAUSED = "paused";
+    static final String NOT_PAUSED = "not_paused";
+    static final String IS_FAVORITE = "is_favorite";
+    static final String NOT_FAVORITE = "not_favorite";
+    static final String HAS_MEDIA = "has_media";
+    static final String NO_MEDIA = "no_media";
+    static final String QUEUED = "queued";
+    static final String NOT_QUEUED = "not_queued";
+    static final String DOWNLOADED = "downloaded";
+    static final String NOT_DOWNLOADED = "not_downloaded";
```

Comment:
I think they should be `public` when you later want to use them in fragments

Reply:
not really sure, i merged the develop into it but it says that its already up to date, what should i do?

Reply:
Maybe your develop branch got some unwanted commits. Hard-reset to the newest version of the upstream develop branch?

```
git remote add upstream git@github.com:AntennaPod/AntennaPod.git
git fetch upstream
git checkout develop
git reset --hard upstream/develop
```

```
git checkout feeditemfilter
git merge develop
```

Reply:
ah i branched my outdated develop, it should work now

Reply:
yea did that, error: `refs/heads/filterids:refs/heads/filterids	[remote rejected] (refusing to allow a Personal Access Token to create or update workflow `.github/workflows/android-emulator.yml` without `workflow` scope)`

Reply:
got it https://stackoverflow.com/questions/64059610/how-to-resolve-refusing-to-allow-an-oauth-app-to-create-or-update-workflow-on